### PR TITLE
Added Linting and Prettier workflow Action

### LINKED ISSUE #11: [ADD]: Addition of Workflows
Add github workflows for linting and prettier.

### PR DESCRIPTION
**Pull request by chetak123**
## Related Issue

- Fixes #11  (issue) 

## Proposed Changes 

I have added [prettier](https://github.com/marketplace/actions/prettier-action) and [Super-Linter ](https://github.com/marketplace/actions/super-linter) Workflow Action

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [master, main]
+    # Remove the line above to run when pushing to master
+  pull_request:
+    branches: [master, main]
```
Comment (AbhijithGanesh) [inline .github/workflows/linter.yml:22]: Please remove `master`

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: .github/workflows/prettier.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+# This action works with pull requests and pushes
+on:
+  pull_request:
```
Comment (AbhijithGanesh) [inline .github/workflows/prettier.yml:5]: can you add branches here?

**Review by AbhijithGanesh — COMMENTED** [review]
Look

**Review by AbhijithGanesh — APPROVED** [review]
Looks Good to Me

**Review by shreemaan-abhishek — COMMENTED** [review]
LGTM as well but how do we make sure that this works as intended.

**Comment by chetak123** [comment]
hey @shreemaan-abhishek the prettier.yml worked here https://github.com/dokc/rap-god-api/pull/20/commits/79339b44b55a218e6d981ad84d94e0342ef5c057

**Comment by shreemaan-abhishek** [comment]
Cool stuff!

**Pull request merged**